octopoes/octopoes/models/ooi/network.py

```diff
@@ -58,11 +83,33 @@ class IPAddressV6(IPAddress):
 
 
 class Protocol(Enum):
+    """Represents the protocol used.
+
+    Possible value
+    -------------
+    tcp, udp
+
+    Example value
+    -------------
+    tcp
+    """
+
     TCP = "tcp"
     UDP = "udp"
 
 
 class PortState(Enum):
+    """Represents the state of the identified ports.
```

Comment:
Add that this is deprecated. Ports that are in KAT are always assumed to be open.